src/components/sidebar/splitLinkView/SplitLinkView.tsx

```diff
@@ -0,0 +1,219 @@
+import { LinkStore } from '~/stores/linkStore';
+import { NodeStore } from '~/stores/nodeStore';
+import RoutePathService from '~/services/routePathService';
+import { ILink, INode, IRoutePath } from '~/models';
+import NodeType from '~/enums/nodeType';
+import LinkService from '~/services/linkService';
+import NodeService from '~/services/nodeService';
+import { NetworkStore } from '~/stores/networkStore';
+import Loader from '~/components/shared/loader/Loader';
+import { Button } from '~/components/controls';
+import ButtonType from '~/enums/buttonType';
+import { AlertStore, AlertType } from '~/stores/alertStore';
+import SidebarHeader from '../SidebarHeader';
+import InputContainer from '../InputContainer';
+import RoutePathSelector from './RoutePathSelector';
+import SplitLinkInfo from './SplitLinkInfo';
+import * as s from './splitLinkView.scss';
+
+interface ISplitLinkViewState {
+    isLoading: boolean;
+    link: ILink | null;
+    node: INode | null;
+    selectedDate?: Date;
+    selectedRoutePathIds: string[];
+    routePaths: IRoutePath[];
+    isLoadingRoutePaths: boolean;
+}
+
+interface ISplitLinkViewProps extends RouteComponentProps<any>{
+    mapStore?: MapStore;
+    errorStore?: ErrorStore;
+    linkStore?: LinkStore;
+    nodeStore?: NodeStore;
+    alertStore?: AlertStore;
+    networkStore?: NetworkStore;
+}
+
+@inject('mapStore', 'errorStore', 'linkStore', 'nodeStore', 'alertStore', 'networkStore')
+@observer
+class SplitLinkView extends React.Component<ISplitLinkViewProps, ISplitLinkViewState> {
+    constructor(props: ISplitLinkViewProps) {
+        super(props);
+        this.state = {
+            isLoading: false,
+            link: null,
+            node: null,
+            selectedRoutePathIds: [],
+            routePaths: [],
+            isLoadingRoutePaths: false,
+        };
+    }
+
+    private init = async () => {
+        this.setState({ isLoading: true });
+
+        const [
+            linkStartNodeId,
+            linkEndNodeId,
+            linkTransitType,
+            nodeId,
+        ] = this.props.match!.params.id.split(',');
+        try {
+            if (linkStartNodeId && linkEndNodeId && linkTransitType && nodeId) {
+                const link = await LinkService.fetchLink(
+                    linkStartNodeId, linkEndNodeId, linkTransitType,
+                );
+                const node = await NodeService.fetchNode(nodeId);
+                this.setState({
+                    link,
+                    node,
+                });
+                this.props.linkStore!.setLink(link);
+                this.props.linkStore!.setIsLinkGeometryEditable(false);
+                this.props.linkStore!.setNodes([node]);
+                const bounds = L.latLngBounds(link.geometry);
+                bounds.extend(node.coordinates);
+                this.props.mapStore!.setMapBounds(bounds);
+                this.props.networkStore!.hideAllMapLayers();
```

Comment:
Is this needed? Now when user goes back from split, he needs to put network nodes and links back on manually.

I wouldn't hide layers (or if I would have to do it, I would put them back on how they were)

Reply:
Gotta test what works best

Reply:
saved previous network layers in store, and reset them when leaving the view